README.md

```diff
@@ -3,6 +3,28 @@ Pi4J :: Java I/O Library for Raspberry Pi :: Devices
 
 Collection of devices implemented for use with Pi4J V2.
 
+
+  __  __ _               _   _          
+|  \/  (_)__ _ _ _ __ _| |_(_)___ _ _  
+| |\/| | / _` | '_/ _` |  _| / _ \ ' \
+|_|  |_|_\__, |_| \__,_|\__|_\___/_||_|
+| |_ ___ |___/                         
+|  _/ _ \                              
+\__\___/ _
+|   \ _ _(_)_ _____ _ _ ___            
+| |) | '_| \ V / -_) '_(_-<            
+|___/|_| |_|\_/\___|_| /__/
+
+We began major changes in the Device Examples.  We are migrating to drivers
+that separate the device example code from external dependencies. These examples will
+be modified one by one. 
+
+Migrated devices are located in https://github.com/Pi4J/pi4j-drivers/tree/main
+
+When a device is migrated its table entry will be updated to indicate that change (M).
```

Comment:
Is this important for users to have here?

- It should be relatively obvious when looking at the corresponding drivers
- It's easy to forget to update and incorrect information seems to be worse than no information

Reply:
PR109